docs/dev-guide/3.0-development/data-modeling.rst

```diff
@@ -0,0 +1,8 @@
+.. warning::
+    All documents within the 3.0-Development section should be considered temporary.
+
+
+Data Modeling
+===============
```

Comment:
2 less `=` to match length of section name  but *really* don't know if it's important.